include/ebpf_program_types.h

```diff
@@ -4,22 +4,15 @@
 
 #include "ebpf_base.h"
 #include "ebpf_result.h"
-
-#include <guiddef.h>
-#if !defined(NO_CRT) && !defined(_NO_CRT_STDIO_INLINE)
-#include <stdint.h>
-#else
-typedef unsigned char uint8_t;
-typedef unsigned int uint32_t;
-typedef unsigned long long uint64_t;
-typedef unsigned short wchar_t;
-#endif
+#include "ebpf_windows.h"
 
 #define EBPF_MAX_PROGRAM_DESCRIPTOR_NAME_LENGTH 256
 #define EBPF_MAX_HELPER_FUNCTION_NAME_LENGTH 256
 
 typedef struct _ebpf_program_type_descriptor
 {
+    ebpf_extension_header_t header;
+    // The following fields are available in version EBPF_PROGRAM_TYPE_DESCRIPTOR_VERSION_0 and later.
```

Comment:
This comment seems incorrect.  "and later" shouldn't apply to versions.  Same in line 26.  I'd replace these comments with comments saying this is the typedef when version = 0.